test/fixtures/users.yml

```
@@ -115,3 +115,14 @@ system_admin:
   auth_source: internal
   password_hash: 02d7ff9921071af778ff4f8608579dcd6d80dfba
   password_salt: 80a167f1effbd82c2485ed81c3cfd68b11bc40dc
+
+external_disabled:
```

I'd rather have enabled user and disable him during our test. Other tests are more likely to use enabled user :)
```suggestion
external:
```